structs.go

```diff
@@ -1170,6 +1170,13 @@ func (m *Member) AvatarURL(size string) string {
 
 }
 
+// BannerURL returns the URL of the member's banner image.
+//    size:    The size of the desired banner image as a power of two
+//             Image size can be any power of two between 16 and 4096.
+func (m *Member) BannerURL(size string) string {
```

Comment:
Not sure if this would be necessary, since `Member` doesn't have `Banner` property

Reply:
I think it would be better to just implement it when it goes out. DiscordGo is 1:1 mapping, so if API doesn't have it, we don't as well.